data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+			label bought
+			action
+				set "Unfettered: bought ship"
+			`	He shows you to the ship you requested and tells you all about its different qualities, but you don't really listen - you only bought this ship so he would let you finish your scans. You did have to spend a fine sum for it though... You could always sell it back to someone else, at the shipyard. Maybe you could use this ship's scanners to get an even better scan of the ships here.`
+			label shipbought
+			`	Now that you're in the newly acquired ship, you pretend to look around until he leaves, seeing you are satisfied with your purchase. Then you get back to scanning the shipyard.`
+				goto finishscan
+			label suspect
+			`	"Oh well, that is indeed a most important thing! Please leave this shipyard, immediately." He says this with slightly more intensity, and you wonder if it's a threat.`
+			choice
+				`	(Pretend to leave, and then come back to finish the scans.)`
+				`	(Leave without the scans.)`
+					decline
+			label finishscan
+			branch toosuspect
+				has "Unfettered: too suspect"
+			`	You manage to finish the scans without any more interruption, and are eager to get back to your ship to get all of this over with.`
+				accept
+			label toosuspect
+			action
+				fine 100000
+			`	It looks like you have aroused too much suspicion. Maybe he was waiting to see if you would actually leave, or maybe he alerted the local authorities, but the result is the same: they arrest you and even fine you <fine>.`
+			action
+				clear "Unfettered: too suspect"
+	on decline
+		fail "Unfettered: Solifuge 6: Shipyard Upgrade"
+
+
+
+mission "Unfettered: Solifuge 8: Useful Korath Scientists"
+	landing
+	name `Primitive Cooling Technology`
+	description `Taer-Fi has asked you to bring coolants to her on <origin>, where she is working with the Korath you rescued to design a better cooling system. Bring back 300 tons of nitrogen human cooling.`
+	source "Firelode"
+	to offer
+		has "Unfettered: Alphas 2: Prove Yourself: done"
+	on offer
+		conversation
+			`As soon as you land, an Unfettered notifies you that Taer-Fi wishes to speak with you, and you should come with him to see her in her labs.`
+			choice
+				`	(Follow him.)`
+				`	"I don't have time right now. I'll come back later."`
+					defer
+			`	The lab is like nothing you've seen before, certainly not on the surface. Here, things are not only efficient but aesthetically pleasing. It almost looks like a museum, with different kinds of alien technologies. You find Taer-Fi working around a Williwaw cooler, but she's not alone. The Korath prisoners you rescued are with her. It appears they have recovered from recent events, although you notice one is missing. They all look around, seeming excited, and like they can't focus on one thing at a time. Their main focus, however, is on the cooler and its surrounding technologies, so they have not noticed you yet.`
```

Comment:
If we're going to explicitly notice one is missing, you should give more specific information about these Korath and how we would even notice one is missing. For instance, I had in mind we found like 20 Korath - how would we notice one is missing from those 20 unless we literally counted?

Reply:
There are not many Koraths btw, but I'll make that one have a singular injury